<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDF_reader</class>
 <widget class="QMainWindow" name="PDF_reader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PDF_reader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/Image/pdf.jpg</normaloff>:/Image/pdf.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_1">
        <attribute name="title">
         <string>目录</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTreeView" name="menu_view">
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>书签</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListView" name="bookmark_view">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QPdfView" name="pdf_view" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen"/>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>页面</string>
    </property>
    <addaction name="actionprevious_page"/>
    <addaction name="actionnext_page"/>
    <addaction name="actionzoom_in"/>
    <addaction name="actionzoom_out"/>
   </widget>
   <widget class="QMenu" name="menutest">
    <property name="title">
     <string>test</string>
    </property>
    <addaction name="actiontest_window"/>
   </widget>
   <widget class="QMenu" name="menubookmark">
    <property name="title">
     <string>书签</string>
    </property>
    <addaction name="actionmark_this_page"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuview"/>
   <addaction name="menubookmark"/>
   <addaction name="menutest"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionopen"/>
   <addaction name="actionprevious_page"/>
   <addaction name="actionnext_page"/>
   <addaction name="actionzoom_out"/>
   <addaction name="actionzoom_in"/>
   <addaction name="actionmark_this_page"/>
  </widget>
  <action name="actionopen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/open.jpg</normaloff>:/Image/open.jpg</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionprevious_page">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/previous.jpg</normaloff>:/Image/previous.jpg</iconset>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </action>
  <action name="actionnext_page">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/next.jpg</normaloff>:/Image/next.jpg</iconset>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </action>
  <action name="actiontest_window">
   <property name="text">
    <string>test_window</string>
   </property>
  </action>
  <action name="actionzoom_in">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/bigger.jpg</normaloff>:/Image/bigger.jpg</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="actionzoom_out">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/littler.jpg</normaloff>:/Image/littler.jpg</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="actionmark_this_page">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/bookmark.jpg</normaloff>:/Image/bookmark.jpg</iconset>
   </property>
   <property name="text">
    <string>书签/取消书签</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPdfView</class>
   <extends>QWidget</extends>
   <header>qpdfview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
